### PR TITLE
[POC] Add classic compiler pass and tagged services

### PR DESCRIPTION
**Pull request by nidup**

### CODE REVIEW

File: DependencyInjection/Compiler/RegisterTaggedJobPass.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+     */
+    public function process(ContainerBuilder $container)
+    {
+        $this->registerConverters($container);
+    }
+
+    /**
+     * @param ContainerBuilder $container
+     */
+    protected function registerConverters(ContainerBuilder $container)
```
Comment (nidup) [inline DependencyInjection/Compiler/RegisterTaggedJobPass.php]: registerJobs

File: DependencyInjection/Compiler/RegisterTaggedJobPass.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    /** @staticvar */
+    const JOB_TAG = 'akeneo_batch.job';
+
+    /**
+     * {@inheritdoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+        $this->registerConverters($container);
```
Comment (nidup) [inline DependencyInjection/Compiler/RegisterTaggedJobPass.php]: registrerJobs

**Comment by juliensnz** [comment]
How will look the config file ?

**Comment by nidup** [comment]
would look like that https://github.com/akeneo/pim-community-dev/pull/2884

once again an old POC i don't say we should do it ;)

**Pull request closed without merging**